### PR TITLE
The new website

### PR DESCRIPTION
**Pull request by juliendonck**
Oh snap. 
Screen Shot 2020-11-30 at 10 45 28

### CODE REVIEW

File: docs/CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-radicle.xyz
```
Comment (xla) [inline docs/CNAME:1]: Is this kosher? Should it not point to radicle.xyz?

File: docs/privacy.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  <link rel="stylesheet" type="text/css" href="styles/style.css" />
+  <meta http-equiv="content-type" content="text/html;charset=utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="twitter:card" content="summary" />
+  <meta name="twitter:site" content="@radicle" />
+  <meta name="twitter:creator" content="@radicle" />
+  <meta property="og:title" content="Radicle - Peer-to-peer code collaboration" />
+  <meta property="og:image" content="https://radicle.xyz/img/social-share.jpg" />
+  <meta property="og:description" content="Radicle is a peer-to-peer stack for building software together." />
+  <meta property="description" content="Radicle is a peer-to-peer stack for building software together." />
```
Comment (NunoAlexandre) [inline docs/privacy.html:15]: We should have this `head` in a separate file that would then be included in every HTML file.

File: docs/styles/typography.css [hunk 3]
```diff
@@ -1,83 +1,91 @@
-@font-face {
-  font-family: "FoundersGrotesk";
-  src: url("../fonts/FoundersGroteskTextWeb-Regular.woff2") format("woff2"), url("../fonts/FoundersGroteskTextWeb-Regular.woff") format("woff"), url("../fonts/FoundersGroteskTextWeb-Regular.eot?") format("eot");
-}
+@import url("https://rsms.me/inter/inter.css");
 
 @font-face {
-  font-family: "FoundersGroteskItalic";
-  src: url("../fonts/FoundersGroteskTextWeb-RegularItalic.woff2") format("woff2"), url("../fonts/FoundersGroteskTextWeb-RegularItalic.woff") format("woff"), url("../fonts/FoundersGroteskTextWeb-RegularItalic.eot?") format("eot");
+  font-family: "Inter";
```
Comment (NunoAlexandre) [inline docs/styles/typography.css:4]: We should offer a fall-back font that looks fairly alike.

File: pages/404.html.mustache [hunk 4]
```diff
@@ -0,0 +1,19 @@
+<!doctype html>
```
Comment (NunoAlexandre) [inline pages/404.html.mustache:1]: :heart: mustache

File: partials/svg-check.mustache [hunk 5]
```diff
@@ -0,0 +1,3 @@
+<svg width="48" height="49" viewBox="0 0 48 49" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M43.6175 9.26093C44.2672 8.36762 44.0697 7.11679 43.1764 6.46711C42.2831 5.81743 41.0323 6.01493 40.3826 6.90824L27.1359 25.1225C26.413 26.1164 24.9732 26.2293 24.1042 25.3603L19.4142 20.6704C18.6332 19.8893 17.3669 19.8893 16.5858 20.6704C15.8048 21.4514 15.8048 22.7178 16.5858 23.4988L21.2757 28.1888C23.8828 30.7958 28.2023 30.4569 30.3708 27.4752L43.6175 9.26093ZM12 14.0846C12 12.98 12.8954 12.0846 14 12.0846H30C31.1046 12.0846 32 11.1892 32 10.0846C32 8.98002 31.1046 8.08459 30 8.08459H14C10.6863 8.08459 8 10.7709 8 14.0846V34.0846C8 37.3983 10.6863 40.0846 14 40.0846H34C37.3137 40.0846 40 37.3983 40 34.0846V26.0846C40 24.98 39.1046 24.0846 38 24.0846C36.8955 24.0846 36 24.98 36 26.0846V34.0846C36 35.1892 35.1046 36.0846 34 36.0846H14C12.8954 36.0846 12 35.1892 12 34.0846V14.0846Z"/>
+</svg>
```
Comment (cloudhead) [inline partials/svg-check.mustache:3]:
```suggestion
</svg>

```

File: pages/downloads.html.mustache [hunk 6]
```diff
@@ -0,0 +1,25 @@
+<!doctype html>
+<html>
+{{> head }}
+<body id="download">
+  {{> header }}
+
+  <div class="content">
+
```
Comment (cloudhead) [inline pages/downloads.html.mustache:8]:
```suggestion
```

File: pages/downloads.html.mustache [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+    <img class="headline-gif gif" src="./img/download-flower.gif" alt="Animation of a flower growing.">
+
```
Comment (cloudhead) [inline pages/downloads.html.mustache:10]:
```suggestion
```

File: pages/downloads.html.mustache [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+    <h1 class="headline druk">Try the beta</h1>
+
```
Comment (cloudhead) [inline pages/downloads.html.mustache:12]:
```suggestion
```

File: pages/downloads.html.mustache [hunk 9]
```diff
@@ -0,0 +1,25 @@
...
+    <div id="text">
+      <h1>Upstream <span>Beta</span></h1>
+      <h2>A desktop client for Radicle.</h2>
+      <p>Check out our <a href="https://docs.radicle.xyz/docs/getting-started/getting-started" target="_blank"><span>Guide to Getting Started</span></a>.</p>
+      <button>{{> svg-mac }} Download for MacOS</button>
+      <button>{{> svg-linux }} Download for Linux</button>
+      <form action="mailto:?subject=Download%20the%20Radicle%20app&amp;body=When%20you%20get%20back%20to%20your%20computer%20&#44;%20download%20the%20app%20here:%20www.radicle.xyz/downloads.html"><button id="email-me">{{> svg-email }} Email me a link</button></form>
+
```
Comment (cloudhead) [inline pages/downloads.html.mustache:20]:
```suggestion
```

File: pages/downloads.html.mustache [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+    </div>
+
```
Comment (cloudhead) [inline pages/downloads.html.mustache:22]:
```suggestion
```

File: pages/downloads.html.mustache [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+    <h1 class="headline druk">Try the beta</h1>
+
+    <div id="text">
+      <h1>Upstream <span>Beta</span></h1>
+      <h2>A desktop client for Radicle.</h2>
+      <p>Check out our <a href="https://docs.radicle.xyz/docs/getting-started/getting-started" target="_blank"><span>Guide to Getting Started</span></a>.</p>
+      <button>{{> svg-mac }} Download for MacOS</button>
+      <button>{{> svg-linux }} Download for Linux</button>
+      <form action="mailto:?subject=Download%20the%20Radicle%20app&amp;body=When%20you%20get%20back%20to%20your%20computer%20&#44;%20download%20the%20app%20here:%20www.radicle.xyz/downloads.html"><button id="email-me">{{> svg-email }} Email me a link</button></form>
```
Comment (xla) [inline pages/downloads.html.mustache:16]:
```suggestion
      <form action="mailto:?subject=Download%20the%20Radicle%20app&amp;body=When%20you%20get%20back%20to%20your%20computer%20&#44;%20download%20the%20app%20here:%20www.radicle.xyz/downloads.html">
        <button id="email-me">{{> svg-email }} Email me a link</button>
      </form>
```

**Review by cloudhead — CHANGES_REQUESTED** [review]
Looks good, just some extra whitespace that should be removed, and files that shouldn't be committed:

* `.DS_Store`
* `docs/node_modules/.yarn-integrity`
* `docs/yarn.lock`

File: partials/svg-user.mustache [hunk 12]
```diff
@@ -0,0 +1,7 @@
+<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (NunoAlexandre) [inline partials/svg-user.mustache:1]: Do these need to be mustache template file? :thinking:

**Review by NunoAlexandre — DISMISSED** [review]
Will now give it a spin locally :runner:

**Review by xla — APPROVED** [review]
🕎 🌮 💰 🌌

**Comment by NunoAlexandre** [comment]
>Looks good, just some extra whitespace that should be removed, and files that shouldn't be committed:
> * `docs/yarn.lock`


@juliendonck @cloudhead `yarn.lock` should be committed [0].

[0]: https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

File: docs/styles/typography.css (hunk 3 above)
Comment (NunoAlexandre) [reply docs/styles/typography.css:4]: Sorry, overlook on my side. Looks good!

File: docs/privacy.html (hunk 2 above)
Comment (NunoAlexandre) [reply docs/privacy.html:15]: Not a blocker btw, something we can do later.

**Comment by cloudhead** [comment]
> > Looks good, just some extra whitespace that should be removed, and files that shouldn't be committed:
> > 
> > * `docs/yarn.lock`
> 
> @juliendonck @cloudhead `yarn.lock` should be committed [0](https://classic.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/).

This isn't a node.js project..

File: docs/CNAME (hunk 1 above)
Comment (cloudhead) [reply docs/CNAME:1]: Was wondering the same :eyes:

File: docs/privacy.html (hunk 2 above)
Comment (cloudhead) [reply docs/privacy.html:15]: It's in `partials/`, unfortunately we have to commit the output too.

File: docs/styles/typography.css (hunk 3 above)
Comment (juliendonck) [reply docs/styles/typography.css:4]: +1

File: partials/svg-user.mustache (hunk 12 above)
Comment (juliendonck) [reply partials/svg-user.mustache:1]: It makes the html a lot more manageable

File: docs/privacy.html (hunk 2 above)
Comment (juliendonck) [reply docs/privacy.html:15]: we do it's the `head.html.mustache`

File: docs/CNAME (hunk 1 above)
Comment (juliendonck) [reply docs/CNAME:1]: weird

**Review by cloudhead — DISMISSED** [review]
:partying_face:

**Review by NunoAlexandre — DISMISSED** [review]
:seedling: What a Radicle website :smiley:

File: partials/svg-user.mustache (hunk 12 above)
Comment (cloudhead) [reply partials/svg-user.mustache:1]: Yeah, to be included :/ afaict

**Review by cloudhead — APPROVED** [review]

**Pull request merged**